lib/Serialization/Serialization.cpp

```diff
@@ -4525,7 +4525,7 @@ void swift::serialize(ModuleOrSourceFile DC,
 
   bool hadError = withOutputFile(getContext(DC), options.OutputPath,
                                  [&](raw_ostream &out) {
-    SharedTimer timer("Serialization (swiftmodule)");
+    SharedTimer timer("Serialization, swiftmodule");
```

Comment:
There is, but LLVM has an assertion that the string is non-escape-requiring. Currently if you run plain-old swift with timers, asserts on, and LLVM's JSON output, it fails.